### PR TITLE
Summer school content changes

### PR DESCRIPTION
**Pull request by korgan00**

### CODE REVIEW

**Review by techtolentino — APPROVED** [review]
looks good, @korgan00

File: new-content/events/summer-school-2021-page/faq.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ linearalgebra:
 qceducators:
   url: https://quantum-computing.ibm.com/programs/educators
   segment: quantum-computing-educators
+mailtoQcEducators:
+  url: mailto:quantum.education@us.ibm.com
+  segment: mailto-quantum-educators
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:22]: We are trying to unify how we do things, so, until we find a better way to do it, please, follow here the same structure as we have on `summer-school-2021.yaml`:

```
segment:
      action: 'events-summer-school > faq > mailto-quantum-educators'
```

Same for all the links. And please, remove the code you don't need

File: new-content/events/summer-school-2021.yaml [hunk 2]
```diff
@@ -13,6 +13,8 @@ header:
   cta:
     label: 'Learn more'
     url: 'https://qiskit.org'
+    segment:
+      action: 'events-summer-school > header > learn-more'
```
Comment (y4izus) [inline new-content/events/summer-school-2021.yaml:17]:
```suggestion
      action: 'events-summer-school > header > qiskit-org-home'
```

File: new-content/events/summer-school-2021.yaml [hunk 3]
```diff
@@ -23,6 +25,8 @@ header:
     time:
     to: 'https://qiskit.org'
     ctaLabel: 'Learn more'
+    segment:
+      action: 'events-summer-school > header > learn-more'
```
Comment (y4izus) [inline new-content/events/summer-school-2021.yaml:29]:
```suggestion
      action: 'events-summer-school > header > qiskit-org-home'
```

File: new-content/events/summer-school-2021.yaml [hunk 4]
```diff
@@ -13,6 +13,8 @@ header:
   cta:
     label: 'Learn more'
     url: 'https://qiskit.org'
```
Comment (y4izus) [inline new-content/events/summer-school-2021.yaml:15]: Ask @JRussellHuffman if this link should have an arrow. If the answer is "yes", replace this url with "/"

File: new-content/events/summer-school-2021.yaml [hunk 5]
```diff
@@ -23,6 +25,8 @@ header:
     time:
     to: 'https://qiskit.org'
```
Comment (y4izus) [inline new-content/events/summer-school-2021.yaml:26]: Ask @JRussellHuffman if this link should have an arrow. If the answer is "yes", replace this url with "/"

File: new-content/events/summer-school-2021.yaml [hunk 6]
```diff
@@ -241,18 +245,26 @@ helpfulResources:
...
         label: 'Enquire now'
+        segment: 
+          action: 'events-summer-school > helpful-resources > qgss2021-enquiry-form'
     - title: 'Learn about and install Qiskit'
       description: 'Prepare for the Summer School by reviewing the documentation and installing Qiskit.'
       cta:
         url: 'https://qiskit.org/documentation/install.html'
         label: 'Install here'
+        segment: 
+          action: 'events-summer-school > helpful-resources > install-here'
```
Comment (y4izus) [inline new-content/events/summer-school-2021.yaml:256]:
```suggestion
          action: 'events-summer-school > helpful-resources > qiskit-install'
```

**Review by y4izus — CHANGES_REQUESTED** [review]
Please, review my suggestions and ask me for another review. I will like to review again the action messages once these changes will be done

File: new-content/events/summer-school-2021-page/faq.md [hunk 7]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 khanacademy:
   url: https://www.khanacademy.org/math/precalculus/x9e81a4f98389efdf:matrices/x9e81a4f98389efdf:multiplying-matrices-by-matrices/v/matrix-multiplication-intro
-  segment: khanacademy-matrix-multiplication-intro
+  segment:
+      action: 'events-summer-school > faq > khanacademy-matrix-multiplication-intro'
 pythonandjupyter:
   url: https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html
-  segment: qiskit-python-and-jupyter-notebooks
+  segment:
+      action: 'events-summer-school > faq > qiskit-python-and-jupyter-notebooks'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:15]:
```suggestion
      action: 'events-summer-school > faq > textbook-python-and-jupyter-notebooks'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 8]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 linearalgebra:
   url: https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:17]: The link says "linear algebra" but it's pointing to the jupyter notebooks. Also, I think the linear algebra section was removed from the textbook. could you check this and change the url in consequence please?

File: new-content/events/summer-school-2021-page/faq.md [hunk 9]
```diff
@@ -7,40 +7,56 @@ module: faq
...
-  segment: qiskit-linear-algebra
+  segment:
+      action: 'events-summer-school > faq > qiskit-linear-algebra'
 qceducators:
   url: https://quantum-computing.ibm.com/programs/educators
-  segment: quantum-computing-educators
+  segment:
+      action: 'events-summer-school > faq > quantum-computing-educators'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:23]:
```suggestion
      action: 'events-summer-school > faq > ibm-quantum-educators-program'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 10]
```diff
@@ -7,40 +7,56 @@ module: faq
...
   url: https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html
-  segment: qiskit-python-and-jupyter-notebooks
+  segment:
+      action: 'events-summer-school > faq > qiskit-python-and-jupyter-notebooks'
 linearalgebra:
   url: https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html
-  segment: qiskit-linear-algebra
+  segment:
+      action: 'events-summer-school > faq > qiskit-linear-algebra'
 qceducators:
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:20]: Just to follow with how you're naming variables:

```suggestion
qCEducators:
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 11]
```diff
@@ -7,40 +7,56 @@ module: faq
...
   url: https://quantum-computing.ibm.com/programs/educators
-  segment: quantum-computing-educators
+  segment:
+      action: 'events-summer-school > faq > quantum-computing-educators'
+mailtoQcEducators:
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:24]:
```suggestion
mailtoQCEducators:
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 12]
```diff
@@ -7,40 +7,56 @@ module: faq
...
+  url: mailto:quantum.education@us.ibm.com
+  segment:
+      action: 'events-summer-school > faq > mailto-quantum-educators'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:27]:
```suggestion
      action: 'events-summer-school > faq > mailto-quantum-educators-program'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 13]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 twitterQiskit:
   url: https://twitter.com/qiskit
-  segment: twitter-qiskit
+  segment:
+      action: 'events-summer-school > faq > twitter-qiskit'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:31]:
```suggestion
      action: 'events-summer-school > faq > qiskit-twitter'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 14]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 twitterIBMResearch:
   url: https://twitter.com/IBMResearch
-  segment: twitter-ibmresearch
+  segment:
+      action: 'events-summer-school > faq > twitter-ibmresearch'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:35]:
```suggestion
      action: 'events-summer-school > faq > ibmresearch-twitter'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 15]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 githubQiskit:
   url: https://github.com/Qiskit
-  segment: github-qiskit
+  segment:
+      action: 'events-summer-school > faq > github-qiskit'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:39]:
```suggestion
      action: 'events-summer-school > faq > qiskit-github'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 16]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 mediumQiskit:
   url: https://medium.com/qiskit
-  segment: medium-qiskit
+  segment:
+      action: 'events-summer-school > faq > medium-qiskit'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:43]:
```suggestion
      action: 'events-summer-school > faq > qiskit-medium'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 17]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 qiskitSubscribe:
   url: https://qisk.it/sub
-  segment: qiskit-subscribe
+  segment:
+      action: 'events-summer-school > faq > qiskit-subscribe'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:47]:
```suggestion
      action: 'events-summer-school > faq > qiskit-youtube-subscribe'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 18]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 joinqiskitslack:
   url: https://ibm.co/joinqiskitslack
-  segment: join-qiskit-slack
+  segment:
+      action: 'events-summer-school > faq > join-qiskit-slack'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:51]:
```suggestion
      action: 'events-summer-school > faq > qiskit-slack-join'
```

File: new-content/events/summer-school-2021-page/faq.md [hunk 19]
```diff
@@ -7,40 +7,56 @@ module: faq
...
 qiskitEvents:
   url: https://qiskit.org/events
-  segment: qiskit-events
+  segment:
+      action: 'events-summer-school > faq > qiskit-events'
```
Comment (y4izus) [inline new-content/events/summer-school-2021-page/faq.md:55]:
```suggestion
      action: 'events-summer-school > faq > qiskit-org-events'
```

File: new-content/events/summer-school-2021.yaml [hunk 20]
```diff
@@ -12,7 +12,9 @@ header:
     - 'Speakers will be announced soon!'
   cta:
     label: 'Learn more'
-    url: 'https://qiskit.org'
+    url: 'http://qisk.it/qgss2021blog_website_source'
+    segment:
+      action: 'events-summer-school > header > qiskit-org-home'
```
Comment (y4izus) [inline new-content/events/summer-school-2021.yaml:17]: If you change the link, you should also change the segment action
```suggestion
      action: 'events-summer-school > header > ibm-research-blog'
```

File: new-content/events/summer-school-2021.yaml [hunk 21]
```diff
@@ -21,8 +23,10 @@ header:
     location: 'Online'
     date: 'July 12 - 23, 2021'
     time:
-    to: 'https://qiskit.org'
+    to: 'http://qisk.it/qgss2021blog_website_source'
     ctaLabel: 'Learn more'
+    segment:
+      action: 'events-summer-school > header > qiskit-org-home'
```
Comment (y4izus) [inline new-content/events/summer-school-2021.yaml:29]:
```suggestion
      action: 'events-summer-school > header > ibm-research-blog'
```

**Review by y4izus — CHANGES_REQUESTED** [review]
Thanks for the previous changes. I have more suggestions.

Also, the "What are the application requirements for the Summer School?" section under FAQ is duplicate.

Ask me for another review (hopefully the last one)

**Review by y4izus — APPROVED** [review]
Thanks!

**Pull request merged**